Sources/Views/MessageContainerView.swift

```diff
@@ -30,6 +30,8 @@ open class MessageContainerView: UIImageView {
 
     private let imageMask = UIImageView()
 
+    open var styleInsets: UIEdgeInsets = .zero
```

Comment:
Is this intended to affect the intrinsic content size of the Container, or just offset the image within the container? (I'm assuming it is the latter, just confirming)

Reply:
Yes the latter - see here it's just insetting the image mask: https://github.com/MessageKit/MessageKit/pull/1249/files#diff-bac966c08f96c77572148721a36fd0f0R54

Reply:
Can you post a screenshot of what this looks like in use? We might also want to add some additional function documentation explaining how to use this and how it differs from `incoming/outgoingMessagePadding`

Reply:
@austinwright just as a side note when I was starting it was confusing to me the amount of padding and insets and how they relate to each other. I think we could use a better documentation. Perhaps I can help with this?

Reply:
Sure you can, we always welcome PRs about documentation, tests and this kind of stuff! :)